### ISSUE TITLE
Don't delete existing `sftp.json` configuration file

### ISSUE THREAD

**Issue by tyuldashev** [UnitTestBot/UTBotCpp#529]
**Description**
In case user leaves empty SFTP port in UTBot quick setup wizard, UTBot removes sftp config file even it was pre-existing.

**To Reproduce**
1. Make sure you already have `.vscode/sftp.json` configuration file
2. Start *UTBot Quickstart* wizard
3. Proceed to 'Server Setup' page
4. Leave `SFTP port` field empty and click 'Next' button

**Actual behavior**
`.vscode/sftp.json` file disappears

**Expected behavior**
Exiting `sftp.json` left as is, because it could be created for reason. For example user wants to control when and what uploaded to the server. It may also contain information and settings which are necessary for user. 

Basically I assume that UBot should not remove anything which wasn't created by the bot itself.

**Comment by tyuldashev** [comment]
The backup with extension `.old` is created but I found bigger problem with current SFTP configuration - #546.
